script/validate-autogen

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
yep I know, all other scripts are `/bin/bash` so.. I did a mimic and was about to do a PR to switch to `/usr/bin/env bash`